src/components/Garden/Filters/TextFilter.tsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react'
 import { GU, SearchInput, textStyle } from '@1hive/1hive-ui'
 
-const TextFilter = React.memo(({ text, onChange, placeholder = '' }) => {
+type TextFilterProps = {
+  text: string
+  placeholder: string
+  onChange: () => void
+}
+
+const TextFilter = ({ text, onChange, placeholder = '' }: TextFilterProps) => {
```

Comment:
You really need placeholder have default to empty string? It's also mean placeholder its optional? Maybe the type definition need be optional too?

Just curisous, not a request of change.

Reply:
Yes sure, it could be optional. Will update it.

Reply:
Updated!